### PR TITLE
GA of dbt Explorer

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

GA of dbt Explorer 

The [Docs Plan](https://www.notion.so/dbtlabs/GA-Explorer-and-all-its-features-9683fd8a5d8e4da58955f89212effba0?pvs=4) for this launch lists all the planned changes.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Needs review from PM and DX

### CODE REVIEW

File: website/docs/docs/collaborate/explore-projects.md [hunk 1]
```diff
@@ -230,7 +228,7 @@ Example of the details view for the model `supplies`:
 
 <Lightbox src="/img/docs/collaborate/dbt-explorer/example-model-details.png" width="100%" title="Example of resource details" />
 
-## Staging environment <Lifecycle status='beta' />
+## Staging environment
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/explore-projects.md:246]: this is ok to remove, right?

File: website/docs/docs/collaborate/explore-projects.md [hunk 2]
```diff
@@ -91,7 +89,7 @@ When you apply a lens, tags become visible on the nodes in the lineage graph, in
 
 - **Default** (resource type)
 - **Materialization Type** (for example, identifying incremental model dependencies)
-- **Lastest Status** (for example, diagnosing a failed DAG region)
+- **Latest Status** (for example, diagnosing a failed DAG region)
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-projects.md]: fixed typo

File: website/docs/docs/collaborate/explore-projects.md [hunk 3]
```diff
@@ -81,20 +81,24 @@ Example of exploring the `order_items` model in the project's lineage graph:
...
+
+- **Relationship**: Organizes resources by resource type, such as models, tests, seeds, and [more](/reference/node-selection/syntax). Resource type uses the `resource_type` selector.
+- **Materialization Type**: Identifies the strategy for building the dbt models in your data platform.
+- **Latest Status**: The status from the latest execution of the resource in the current environment. For example, diagnosing a failed DAG region.
+- **Model Layer**: The modeling layer that the model belongs to according to [best practices guide](https://docs.getdbt.com/best-practices/how-we-structure/1-guide-overview#guide-structure-overview). For example, discovering marts models to analyze.
     - **Marts** &mdash; A model with the prefix `fct_` or `dim_` or a model that lives in the `/marts/` subdirectory.
     - **Intermediate** &mdash; A model with the prefix `int_`. Or, a model that lives in the `/int/` or `/intermediate/` subdirectory.
     - **Staging** &mdash; A model with the prefix `stg_`. Or, a model that lives in the `/staging/` subdirectory.
-
+- **Test Status**: The status from the latest execution of the tests that ran again this resource.
```
Comment (cafzal) [inline website/docs/docs/collaborate/explore-projects.md]: Please add: "In the case that a model has multiple tests with different results, the lens reflects the 'worst case' status. "

File: website/docs/docs/collaborate/column-level-lineage.md [hunk 4]
```diff
@@ -3,8 +3,6 @@ title: "Column-level lineage"
 description: "Use dbt Explorer's column-level lineage to gain insights about your data at a granular level."
 ---
 
-# Column-level lineage <Lifecycle status='public preview' />
-
 dbt Explorer now offers column-level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a resource (model, source, or snapshot) in a dbt project, Explorer provides end-to-end lineage for the data in that column given how it's used.
 
 CLL is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the [Discovery API](/docs/dbt-cloud-apis/discovery-api).
```
Comment (cafzal) [inline website/docs/docs/collaborate/column-level-lineage.md]: Please remove "It’s also available through the [Discovery API](/docs/dbt-cloud-apis/discovery-api)."

File: website/docs/docs/collaborate/column-level-lineage.md [hunk 5]
```diff
@@ -20,6 +18,19 @@ dbt Cloud updates the lineage in Explorer after each run that's executed in the
 
 <LoomVideo id='3040bf2a2ade45eca7942a7aed6b730c' />
```
Comment (cafzal) [inline website/docs/docs/collaborate/column-level-lineage.md]: Please remove, now outdated

Comment (runleonarun) [inline website/docs/docs/collaborate/column-level-lineage.md]:
```suggestion
```

File: website/docs/docs/collaborate/column-level-lineage.md (hunk 4 above)
Comment (runleonarun) [inline website/docs/docs/collaborate/column-level-lineage.md]:
```suggestion
CLL is available to dbt Cloud Enterprise accounts that can use Explorer. 
```

Comment (runleonarun) [reply website/docs/docs/collaborate/column-level-lineage.md]: Done! it's included in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/5488/commits/6d409449d1b65a3b892200dc1c5a5753c0522eee)

File: website/docs/docs/collaborate/column-level-lineage.md (hunk 5 above)
Comment (runleonarun) [reply website/docs/docs/collaborate/column-level-lineage.md]: Done! it's included in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/5488/commits/6d409449d1b65a3b892200dc1c5a5753c0522eee)

File: website/docs/docs/collaborate/explore-projects.md [hunk 6]
```diff
@@ -83,28 +81,36 @@ Example of exploring the `order_items` model in the project's lineage graph:
...
+
+- **Relationship**: Organizes resources by resource type, such as models, tests, seeds, and [more](/reference/node-selection/syntax). Resource type uses the `resource_type` selector.
+- **Materialization Type**: Identifies the strategy for building the dbt models in your data platform.
+- **Latest Status**: The status from the latest execution of the resource in the current environment. For example, diagnosing a failed DAG region.
+- **Model Layer**: The modeling layer that the model belongs to according to [best practices guide](https://docs.getdbt.com/best-practices/how-we-structure/1-guide-overview#guide-structure-overview). For example, discovering marts models to analyze.
     - **Marts** &mdash; A model with the prefix `fct_` or `dim_` or a model that lives in the `/marts/` subdirectory.
     - **Intermediate** &mdash; A model with the prefix `int_`. Or, a model that lives in the `/int/` or `/intermediate/` subdirectory.
     - **Staging** &mdash; A model with the prefix `stg_`. Or, a model that lives in the `/staging/` subdirectory.
+- **Test Status**: The status from the latest execution of the tests that ran again this resource.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/explore-projects.md]:
```suggestion
- **Test Status**: The status from the latest execution of the tests that ran again this resource. In the case that a model has multiple tests with different results, the lens reflects the 'worst case' status. 
```

File: website/docs/docs/collaborate/explore-projects.md (hunk 3 above)
Comment (runleonarun) [reply website/docs/docs/collaborate/explore-projects.md]: Added! It's in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/5488/commits/3b01d924416213999f36a06f459c40d28af3ebc9).

File: website/docs/docs/collaborate/explore-projects.md [hunk 7]
```diff
@@ -247,6 +247,8 @@ Example of the details view for the model `supplies`:
 
 dbt Explorer supports views for [Staging deployment environments](/docs/deploy/deploy-environments#staging-environment), in addition to the Production environment. This gives you a unique view into your pre-production data workflows, with the same tools available in production, while providing an extra layer of scrutiny. Once the Staging environment is configured and has a successful job run, it will be visible on the dbt Explorer landing page.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/explore-projects.md]:
```suggestion
dbt Explorer supports views for [staging deployment environments](/docs/deploy/deploy-environments#staging-environment), in addition to the production environment. This gives you a unique view into your pre-production data workflows, with the same tools available in production, while providing an extra layer of scrutiny. 
```

File: website/docs/docs/collaborate/explore-projects.md [hunk 8]
```diff
@@ -247,6 +247,8 @@ Example of the details view for the model `supplies`:
...
 
+You can explore the metadata from your production or staging environment to inform your data development lifecycle. Just [set a single environment](/docs/deploy/deploy-environments) per dbt Cloud project as “production” or “staging”, and ensure the proper metadata has been generated then you’ll be able to view it in Explorer. Refer to [Generating metadata](/docs/collaborate/explore-projects#generate-metadata) for more details.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/explore-projects.md]:
```suggestion
You can explore the metadata from your production or staging environment to inform your data development lifecycle. Just [set a single environment](/docs/deploy/deploy-environments) per dbt Cloud project as “production” or “staging," and ensure the proper metadata has been generated then you’ll be able to view it in Explorer. Refer to [Generating metadata](/docs/collaborate/explore-projects#generate-metadata) for more details.
```

**Review by runleonarun — APPROVED** [review]
Approved by Cameron and me!

**Pull request merged**